crates/aptos-crypto/benches/slh_dsa_sha2_128s.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Aptos Foundation
+// Licensed pursuant to the Innovation-Enabling Source Code License, available at https://github.com/aptos-labs/aptos-core/blob/main/LICENSE
+
+#[macro_use]
+extern crate criterion;
+
+use aptos_crypto::{
+    slh_dsa_sha2_128s::{PrivateKey, PublicKey, Signature},
+    traits::{Signature as SignatureTrait, SigningKey, Uniform},
+    PrivateKey as PrivateKeyTrait,
+};
+use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use criterion::{measurement::Measurement, BenchmarkGroup, Criterion, Throughput};
+use rand::{thread_rng, RngCore};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, CryptoHasher, BCSCryptoHash, Serialize, Deserialize)]
+pub struct Message32Bytes([u8; 32]);
+
+impl Message32Bytes {
+    fn new(bytes: [u8; 32]) -> Self {
+        Message32Bytes(bytes)
+    }
+}
+
+fn benchmark_groups(c: &mut Criterion) {
+    let mut group = c.benchmark_group("slh_dsa/sha2-128s");
+
+    group.sample_size(10);
+
+    sig_deserialize(&mut group);
+    pk_deserialize(&mut group);
+    sign_32_bytes(&mut group);
+    verify_32_bytes(&mut group);
+
+    group.finish();
+}
+
+/// Benchmarks the time to deserialize an SLH-DSA signature from a sequence of bytes.
+fn sig_deserialize<M: Measurement>(g: &mut BenchmarkGroup<M>) {
+    let mut csprng = thread_rng();
+
+    g.throughput(Throughput::Elements(1_u64));
+    g.bench_function("sig_deserialize", move |b| {
+        b.iter_with_setup(
+            || {
+                let priv_key = PrivateKey::generate(&mut csprng);
+                let mut msg_bytes = [0u8; 32];
```

Comment:
nit: could you create a helper function that you use across benchmark steps? e.g.,

```
fn generate_random_message(csprng: ...) {
    let mut msg_bytes = [0u8; 32];
    csprng.fill_bytes(&mut msg_bytes);
    Message32Bytes::new(msg_bytes)
}
```

Reply:
Fixed!